index.html

```diff
@@ -5,8 +5,149 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+    <header class="page-header">
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigtion-link" href="about.html">Главная</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigtion-link" href="sedona.html">О Седоне</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigtion-link" href="hotels.html">Гостиницы</a>
```

Comment:
```suggestion
            <a class="navigtion-link" href="catalog.html">Гостиницы</a>
```
Поскольку файл внутренней страницы вы назвали **catalog.html** (иначе у нас нет ссылки на неё).